src/main/resources/avro/bdg.avdl

```diff
@@ -1221,4 +1221,166 @@ record Sample {
    */
   map<string> attributes = {};
 }
-}
\ No newline at end of file
+
+/**
+ Alphabet.
+ */
+enum Alphabet {
+
+  /**
+   DNA alphabet.
+   */
+  DNA,
+
+  /**
+   RNA alphabet.
+   */
+  RNA,
+
+  /**
+   Protein alphabet.
+   */
+  PROTEIN
+}
+
+/**
+ Sequence.
```

Comment:
More concretely, this is a record that describes a DNA contig, or an RNA transcript, or a protein amino acid sequence, right? If so, the docs should say this.